lib/discourse_data_explorer/result_to_markdown.rb

```diff
@@ -4,7 +4,7 @@
 
 module ::DiscourseDataExplorer
   class ResultToMarkdown
-    def self.convert(pg_result)
+    def self.convert(pg_result, opts = {})
```

Comment:
Wondering, since we're passing only one option, and it's done explicitly, if we should use a keyword argument instead of an options hash here?

Reply:
Oh yeah I meant to say, I wasn't really sure if `render_url_columns` was too specific or if we should have some generic `render_special_columns: true|false` option. But yes maybe for now we can just start with the keyword arg here